manifests/templates/opensearch/test/manifest.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Any test data that you need to add can be added under https://github.com/opensearch-project/opensearch-build/tree/main/tests/data

Reply:
I have moved the test data, thanks!